packages/expo-modules-core/ios/Tests/ExpoModulesSpec.swift

```diff
@@ -61,7 +67,7 @@ class ExpoModulesSpec: ExpoSpec {
         let dict = try runtime.eval("expo.modules.TestModule").asDict()
 
         dict.forEach { (key: String, value: Any) in
-          expect(value) === dict[key]!
+          expect((value as! NSObject) === dict[key]! as! NSObject).to(beTrue())
```

Comment:
Same comment about having the assertion library do the equality check

Reply:
done